MAUI/DataForm/Validation.md

```diff
@@ -15,6 +15,115 @@ The data form validates the data and user input to update the correct value in t
 
 The supported built in validations are as follows:
 
+#### IDataErrorInfo
+You can validate the data by implementing the [IDataErrorInfo](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.idataerrorinfo?view=net-6.0) interface in the data object class.
+
+{% tabs %}
+{% highlight C# %}
+
+public class EmployeeInfo : IDataErrorInfo
+{
+    private int employeeID;
+    private string name;
+    private string title;       
+
+    public event PropertyChangedEventHandler PropertyChanged;
```

Comment:
private int employeeID;
    private string name;
    private string title;       

    public event PropertyChangedEventHandler PropertyChanged;


    public EmployeeInfo()
    {

    }

remove if not needed

Reply:
private fields removed